administration/src/bp-modules/regions/data-privacy-policy/DataPrivacyOverview.tsx

```diff
@@ -39,7 +39,7 @@ type RegionOverviewProps = {
 
 const MAX_CHARS = 20000
 
-const RegionOverview = ({ dataPrivacyPolicy, regionId }: RegionOverviewProps): ReactElement => {
+const DataPrivacyOverview = ({ dataPrivacyPolicy, regionId }: RegionOverviewProps): ReactElement => {
```

Comment:
I am missing a back button in this screen :)

Reply:
ok i though the browser back and the navigation item is sufficient but i can add it